CalibTracker/SiStripDCS/python/dcs_print_summary_cfg.py

```diff
@@ -0,0 +1,41 @@
+import FWCore.ParameterSet.Config as cms
+import os
+
+process = cms.Process("summary")
+
+process.MessageLogger = cms.Service( "MessageLogger",
+                                     debugModules = cms.untracked.vstring( "*" ),
+                                     cout = cms.untracked.PSet( threshold = cms.untracked.string( "DEBUG" ) ),
+                                     destinations = cms.untracked.vstring( "cout" )
+                                     )
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+process.source = cms.Source("EmptySource",
+    numberEventsInRun = cms.untracked.uint32(1),
+    firstRun = cms.untracked.uint32(1)
+)
+
+process.SiStripDetInfoFileReader = cms.Service("SiStripDetInfoFileReader")
+
+process.load("CondCore.CondDB.CondDB_cfi")
+process.load("CalibTracker.SiStripDCS.siStripDetVOffPrinter_cfi")
```

Comment:
I don't see this file in the corresponding path. Is this correct?

Reply:
yes, that's the whole point of adding a `fillDescriptions` method: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Converting_an_Existing_CFI_file

Reply:
I see. Thanks.